source/ocsp-support/ocsp-support.rst

```diff
@@ -211,10 +222,22 @@ This boolean option determines whether a MongoClient should refrain
 checking certificate revocation status. When set to true, a driver
 MUST NOT check certificate revocation status via CRLs or OCSP.  When
 set to false, a driver MUST check certificate revocation status, reach
-out to OCSP endpoints if needed (as described in 
-`Specification: Suggested OCSP Behavior <ocsp-support.rst#id1>`__).  
-This option MUST default to false.
+out to OCSP endpoints if needed (as described in
+`Specification: Suggested OCSP Behavior <ocsp-support.rst#id1>`__).
 
+For drivers that pass the `"Soft Fail Test"
+<tests/README.rst#integration-tests-permutations-to-be-tested>`__ , this
+option MUST default to false.
+
+If a driver has not already defaulted `tlsDisableOCSPEndpointCheck` to
```

Comment:
It might be better to say "If a driver does not support tlsDisableEndpointCheck". As written this might be misunderstood to imply that an implementer can arbitrarily pick which of the options to default to true, but I don't think that's your intent. Is it?

Reply:
That's not my intent. I updated the phrasing in the new commit.